devfiles/cpp/devfile.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: 1.0.0
+metadata:
+  generateName: cpp-
+projects:
+  -
+    name: cpp-hello-world
+    source:
+      type: git
+      location: 'https://github.com/che-samples/cpp-hello-world'
+components:
+-
+  type: chePlugin
+  id: che-incubator/cpptools/latest
+  alias: cpp-plugin
+  preferences:
+    clangd.path: /usr/bin/clangd
+    cdt.clangd.binaries.enable: false
+-
+  type: dockerimage
+  alias: cpp-dev
+  image: quay.io/eclipse/che-cpp7-centos:nightly
+  command: ['sleep']
+  args: ['infinity']
+  memoryLimit: 512Mi
+  mountSources: true
+commands:
+  -
+    name: build
+    actions:
+      - type: exec
+        component: cpp-dev
+        command: g++ -g hello.cpp -o hello.out
```

Comment:
Consider adding `&& echo "Build complete"` to the end here, since compiles produce no output on success and it might be confusing to have no output in the panel.

Reply:
Make sense